controllers/seedimage_controller.go

```diff
@@ -539,6 +508,57 @@ func fillBuildImagePod(name, namespace, buildImg, baseImg, configMap string, pul
 	return pod
 }
 
+func defineInitContainers(baseImg, buildImg string, pullPolicy corev1.PullPolicy) []corev1.Container {
+	const baseIsoPath = "/iso/base.iso"
+
+	containers := []corev1.Container{}
+	buildCommands := []string{
+		fmt.Sprintf("xorriso -indev %s -outdev /iso/elemental.iso -map /overlay / -boot_image any replay", baseIsoPath),
```

Comment:
The iso is downloaded to the shared volume, so no resources additional resources are required

Reply:
We should clean it up then when done, otherwise we will occupy extra disk space on the node

Reply:
I was wondering about it too, but I could not see much benefit out of it, as up to my understanding the size reserved by the pod regardless being used or not. Hence to free resources the pods needs to be terminated. It's true however that after terminating the pod if the volume is smaller then is less likely to require k8s to start cleaning unused resources.

We can add a clean up call here, that's easy.